homeassistant/components/vicare/binary_sensor.py

```diff
@@ -71,6 +72,15 @@ class ViCareBinarySensorEntityDescription(
     ),
 )
 
+SOLAR_SENSORS: tuple[ViCareBinarySensorEntityDescription, ...] = (
```

Comment:
could you please rename COMPRESSOR_SENSORS to GLOBAL_SENSORS and then add this sensor to co GLOBAL_SENSORS?
we don't separate the sensors by heating device type but by how they are structured on PyVicare.